PayloadPkg/OsLoader/OsLoader.h

```diff
@@ -170,6 +171,18 @@ DumpMbInfo (
   IN  CONST MULTIBOOT_INFO *Mi
   );
 
+/**
+Print out the Multiboot-2 information block.
+
+@param[in]  Mi  The Multiboot-2 information block to be printed.
+
+**/
+struct multiboot2_start_tag;    /* XXX: move to header file */
+VOID
+DumpMb2Info (
+  IN  CONST struct multiboot2_start_tag *Mi
+  );
```

Comment:
Add DumpMb2Info  to MultibootLib.h if it is expected to be called by other modules.

Reply:
OsLoader.h already has DumpMbInfo(); should I move that as well?

Reply:
Got it. It is ok to put it together with DumpMbInfo(). Late we could have a separate patch to move them.